modules/ppcp-wc-gateway/src/Helper/MerchantDetails.php

```diff
@@ -13,6 +13,34 @@
  * Main information source about merchant details.
  */
 class MerchantDetails {
```

Comment:
I'm 100% for having a centralized place for this data, but we need to make sure to update everything to use it and also stick to it as the standard/convention.

We need to use the same format and conventions for things like:
- Merchant eligibilities (e.g., local country/currency matrix, etc.)
- Merchant capabilities (e.g., PayPal SellerStatus "features" returned from the API)

Otherwise, we risk fragmentation or confusion in the future.